geopandas/geoseries.py

```diff
@@ -156,7 +204,8 @@ def __finalize__(self, other, method=None, **kwargs):
         """ propagate metadata from other to self """
         # NOTE: backported from pandas master (upcoming v0.13)
         for name in self._metadata:
-            object.__setattr__(self, name, getattr(other, name, None))
+            if not hasattr(self, name):
+                object.__setattr__(self, name, getattr(other, name, None))
```

Comment:
I am not sure the `if not hasattr(self, name):` is correct? 

(but might we can remove this altogether, as this is the same implementation is in pandas itself)